{"name": "test_add[ni-3-error]", "status": "passed", "parameters": [{"name": "a", "value": "'ni'"}, {"name": "b", "value": "3"}, {"name": "expecter", "value": "'error'"}], "start": 1615047945293, "stop": 1615047945294, "uuid": "54468a64-5c2d-43e7-933c-5417c33de816", "historyId": "8217f036d2d610d7189dec24013d2e2a", "testCaseId": "46313a82a5980d1f654f624e59d2d766", "fullName": "test_work_two.TestCalc#test_add", "labels": [{"name": "tag", "value": "first"}, {"name": "tag", "value": "run(order=0)"}, {"name": "suite", "value": "test_work_two"}, {"name": "subSuite", "value": "TestCalc"}, {"name": "host", "value": "192.168.0.102"}, {"name": "thread", "value": "4494-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_work_two"}]}